server/db/schema/degreeprogrammes.sql

```diff
@@ -0,0 +1,109 @@
+-- ------------------------------
+-- Degree Programme Meta Tables
+-- ------------------------------
+
+CREATE TABLE abschlussart (
+    id TEXT PRIMARY KEY,
+    name TEXT
```

Comment:
```suggestion
    name TEXT NOT NULL
```
Sollte nicht null sein dürfen, oder?